cmd/sops/subcommand/updatekeys/updatekeys.go

```diff
@@ -0,0 +1,198 @@
+package updatekeys
+
+import (
+	"fmt"
+
```

Comment:
nit: gofmt

Reply:
Hmm, I don't know how this happens, but `goimports` doesn't seem to care about it. I'll fix it manually.